.gitignore

```diff
@@ -21,3 +21,20 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+# Eclipse
+.classpath
+.project
+.settings/
+
+# Intellij
+.idea/
+*.iml
+*.iws
+
+# Mac
+.DS_Store
+
+# Maven
+log/
```

Comment:
You may want to add the maven wrapper files into this exclusion as well, I'm not sure you'd want to add them to github, as they're for bash/cmd, and everyone should have maven installed anyways. If you have a good argument to keep them, feel free.

Reply:
I assumed that having mvnw scripts will help align the maven version in use with this project's version, but you make a good point. I'll add them to gitignore and then specify the maven version in the Readme instead.